cmd/cli/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/kubescape/go-logger"
+
+func main() {
+	logger.L().Error("CLI not yet implemented")
```

Comment:
issue: `Error()` will exit with return code 0.

But if the CLI is not implemented, I think it constitutes an non-successful execution of the application, which, in my opinion, should return a non-successful exit code.

How about:
```suggestion
	logger.L().Fatal("CLI not yet implemented")
```